.github/workflows/combine-prs.yml

```diff
@@ -0,0 +1,163 @@
+#
+# Combine/group together PRs
+#
+# https://github.com/hrvey/combine-prs-workflow
+#
+
+name: 'Combine PRs'
+
+# Controls when the action will run - in this case triggered manually
+on:
+  workflow_dispatch:
+    inputs:
+      branchPrefix:
+        description: 'Branch prefix to find combinable PRs based on'
+        required: true
+        default: 'dependabot'
+      mustBeGreen:
+        description: 'Only combine PRs that are green (status is success)'
+        required: true
+        default: true
+      combineBranchName:
+        description: 'Name of the branch to combine PRs into'
+        required: true
+        default: 'combine-prs-branch'
+      ignoreLabel:
+        description: 'Exclude PRs with this label'
+        required: true
+        default: 'nocombine'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "combine-prs"
+  combine-prs:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/github-script@v6
+        id: create-combined-pr
+        name: Create Combined PR
+        with:
+          github-token: ${{secrets.GH_TOKEN}}
```

Comment:
You would have to supply your own every time you run it? Or do we populate it with default value with secret (if possible) and expose the value in the UI?

Reply:
Every repo that uses it would have to supply its own token. Like we're doing with `ci-standard-checks` in [here](https://github.com/Typeform/.github/blob/main/.github/workflows/ci-standard-checks.yml#L24). This way we can use [the built-in `GITHUB_TOKEN` that has the scopes limited to the repo itself](https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/) and not this organization wide (`GH_TOKEN`). Does it make sense?

Reply:
Thanks, makes sense 👍 I changed it to `GITHUB_TOKEN`.